src/Nancy.Swagger/Services/RouteUtils/SwaggerRouteDescriber.cs

```diff
@@ -77,6 +77,10 @@ public void AddAdditionalModels(params Type[] types)
                             {
                                 param.Type = "string";
                             }
+                            if(param.Type == "file")
+                            {
+                                op.ConsumeMimeType("multipart/form-data");
```

Comment:
The specs say it has to be either or both. I think I will add both, because I don't see the harm.